amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/ShardConsumerSubscriber.java

```diff
@@ -54,14 +55,22 @@
     @Getter(AccessLevel.PACKAGE)
     private volatile Throwable retrievalFailure;
 
+    @Deprecated
     ShardConsumerSubscriber(RecordsPublisher recordsPublisher, ExecutorService executorService, int bufferSize,
```

Comment:
No need to chain this constructor since it is package private.

Reply:
It's not an internal class, so I wasn't sure if we were concerned about breaking forks that might have added a class that depends on this constructor.